### ISSUE TITLE
Custom ZIM favicon

### ISSUE THREAD

**Issue by kelson42** [openzim/youtube#65]
It should be able to specify a custom ZIM favicon (path/URL)

**Comment by rgaudin** [comment]
I don't think that's necessary. I can't think of a reasonnabe scenario where you'd want to specify a different favicon than the profile image.

We already have a good number of options ; I think customizing profile is good enough for most case.

For skin-of-color for instance, we could define a profile image (they are using a very generic one) and that would serve both. See https://www.youtube.com/channel/UCkPCACzHzFqP1elI5KGMdwg

**Comment by kelson42** [comment]
@rgaudin What is a profile image? Is that the same as ZIM favicon?

**Comment by rgaudin** [comment]
A youtube channel has two images : a large wide `banner` and a square `profile` image. We are reusing the profile image for the favicon as well.
